SuperBuild/External_python.cmake

```diff
@@ -87,9 +91,9 @@ if((NOT DEFINED PYTHON_INCLUDE_DIR
   if(WIN32)
     list(APPEND EXTERNAL_PROJECT_OPTIONAL_CMAKE_ARGS
         # Depends on libbz2
-        -DBUILTIN_BZ2:BOOL=OFF
+        -DBUILTIN_BZ2:BOOL=ON
```

Comment:
Comment above indicates the following:

>   Force modules that statically link to zlib or libbz2 to not be built-in.
>   Otherwise, when building in Debug configuration, the Python library--which
>   we force to build in Release configuration--would mix Debug and Release C
>   runtime libraries.

Unless we confirm it is not an issue anymore, we will likely have to build them as shared.

Reply:
I confirmed that keeping the original option still allow to compile CPython and import `binascii` and `bz2` modules. I am then reverting this changes back to their original value.